filebeat/docs/command-line.asciidoc

```diff
@@ -1,9 +1,15 @@
 [[filebeat-command-line]]
 === Command Line Options
 
-Filebeat does not have any Filebeat-specific command line options. Instead, you
+Filebeat does only have one Filebeat-specific command line option. For all other options you
 configure the behaviour of Filebeat by specifying options in the configuration file.
 
+*`-once`*::
+When the `-once` flag is used, filebeat starts all configured harvesters and prospectors and runs
```

Comment:
When the `-once` flag is used, Filebeat starts all configured harvesters and prospectors, and runs... (capitalize Filebeat and add comma).